app/modules/project/routes.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2014-present PlatformIO <contact@platformio.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ProjectSettingsPage } from '@project/containers/configuration';
+import { ProjectsListPage } from '@project/containers/list';
+
+const routes = [
+  {
+    path: '/projects',
+    component: ProjectsListPage
+  },
+  {
+    path: '/projects/settings',
```

Comment:
How about to call this "configuration" instead of "settings"? We call this "config". Maybe, we will other real SETTINGS. This is a configuration based on `platformio.ini`. Just to keep everything within the same naming scope.